---
layout: about
title: about
permalink: /
subtitle: 

profile:
  align: right
  image: prof_pic.jpg
  image_circular: false # crops the image to make it circular
  more_info: >

news: false # includes a list of news items
selected_papers: true # includes a list of papers marked as "selected={true}"
social: true # includes social icons at the bottom of the page
---

My research focuses on developing bioinformatic software for the genomic epidemiology of infectious disease, especially viruses. I am particularly interested in using metagenomic sequence data for surveillance, identifying the relevant signals over time or space while controlling for the noise. Another interest is in supporting comparative analysis for public health in contexts where data sharing may be limited.

During the SARS-CoV-2 pandemic, I developed and maintained several of the core analysis pipelines for the COVID-19 Genomics UK Consortium (COG-UK). These centrally processed submitted data, providing QC, alignment, variant and lineage calling and built a phylogenetic tree to place them into context within the UK and globally. In addition I worked on software Scorpio and Pangolin which provided variant/constellation and lineage classification.

I currently work at the University of Edinburgh with Andrew Rambaut and as part of the ARTIC Network.
